Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:latest
+
+RUN pip install pyserial flask
```

Comment:
Might be better to have this be built from the `requirements.txt`?

Reply:
My local env is python 3.6, but latest python is 3.9. Not sure how all the deps from py3.6 are affected by 3.9.